tests/performance/apdex-score/README.md

```diff
@@ -214,14 +247,18 @@
 
 </details>
 
-- Find the android version by running `adb shell getprop | grep ro.build.version.release`
-- Find the device name by running `adb shell getprop | grep ro.product.model`
-
-4. Set the environment variable `APDEX_TEST_SETTINGS` with the path of your settings file.
-
-```sh {"id":"01J2WE5FDN0D40ZJ5XA7ZHHH4Z"}
-export APDEX_TEST_SETTINGS=/Users/pepe/Documents/apdex-settings.json
-```
+4. Set the environment variable `APDEX_TEST_SETTINGS` with the path of your settings file (apdex-settings.json). 
+    For example, you can use the following command but make sure to replace the path with your actual settings file location:
+    ```
+    export APDEX_TEST_SETTINGS=/Users/pepe/Documents/apdex-settings.json
+    ```
+   - Ensure the `apdex-settings.json` file has been updated with the correct instance url, login credentials and assertion texts for page navigation, forms and other app interactions.
```

Comment:
Can you please provide more clarity about the test data? The assertion texts should correspond to the data in your instance.